CONTRIBUTING.md

```diff
@@ -111,3 +111,50 @@ Learn by doing and get your hands dirty!
 [f-f]: https://github.com/f-f
 [discord]: https://purescript.org/chat
 [spago-issues]: https://github.com/purescript/spago/issues
+
+## Working with file paths
```

Comment:
I'd like this documentation to exist, but I'm not sure this is the right place for it and I couldn't find a better one.